router/image/templates/nginx.conf

```diff
@@ -15,43 +15,47 @@ http {
     sendfile on;
     tcp_nopush on;
     tcp_nodelay on;
-
     # The Timeout value must be greater than the front facing load balancers timeout value.
     # Default is the deis recommended timeout value for ELB - 1200 seconds + 100s extra.
-    {{ $defaultTimeout := or (.deis_router_defaultTimeout) "1300" }}
```

Comment:
> (I hate the if exists change)

Me too. Maybe we can make this easier on ourselves by introducing a function like https://github.com/kelseyhightower/memkv/pull/4 into our confd fork, which would make the entire template **much** easier to grok. @mboersma @carmstrong would you be okay with this?